examples/gocean/eg4/Makefile

```diff
@@ -69,10 +77,11 @@ psy_gen.f90 alg_gen.f90: openacc
 
 %.o: %.f90
 	# Currently only compatible with F90=pgf90
-	${F90} -Minfo -acc -ta=tesla:cc70 ${LDFLAGS} -I${INF_INC} -c $<
+	${F90} ${F90FLAGS} -I${INF_INC} -c $<
 
 compile-acc: ${INF_LIB} alg_gen.f90 ${KERNELS} psy_gen.o
-	${F90} -acc ${LDFLAGS} -I${INF_INC} alg_gen.f90 psy_gen.o ${INF_LIB} -o alg_openacc.exe
+	${F90} ${F90FLAGS} ${LDFLAGS} -I${INF_INC} alg_gen.f90 psy_gen.o \
```

Comment:
That line was completely messed up. alg_gen.f90? And there was a `${kernels}` in the dependency line, but then not linked in?
I've fixed that all (including linking with `${KERNEL}`, since as far as I could see it also needs `data_mod.o`. I've also added the initialisation, following the example you pointed out. At least on CPU it now compiles, links and executes with address-sanitizer (only reporting the leaked fld1 at the end).

I also noticed that the line:
```
F90FLAGS ?= -Minfo -acc -ta=tesla:cc70
```
in the Makefile is useless (`F90FLAGS` is set in `common.mk`, so it is always defined, so this statement will not do anything). But I wasn't sure if you wanted me to remove it, or maybe move it into the comment?

Reply:
Good spot. Could you move it into the comment and change `?=` to `=`?

Reply:
Done.